src/version.ts

```diff
@@ -0,0 +1,9 @@
+import { SDK_VERSION as SENTRY_REACT_NATIVE_VERSION } from '@sentry/react-native';
+import { SDK_VERSION as SENTRY_REACT_VERSION } from '@sentry/react';
+
+export const SENTRY_EXPO_PACKAGE = 'npm:sentry-expo';
+export const SENTRY_EXPO_VERSION = '6.1.1';
```

Comment:
Can we import this from package.json so we don't have to manually keep this in sync?

Reply:
Hi, I would like not to import the `package.json` here to keep the file out of customers apps (not to impact size).

But I understand the inconvenience of updating extra place. Would [this script](https://github.com/expo/sentry-expo/pull/331/commits/d4ddcd8d337d9f246c10f526a0d9e19d3e602837) be acceptable compromise to easily update the package name and version?